src/test/java/com/wikia/webdriver/common/enums/Mercury.java

```diff
@@ -0,0 +1,31 @@
+package com.wikia.webdriver.common.enums;
+
+/**
+ * @ownership Content X-Wing Wikia
+ */
+public class Mercury {
+
+  /**
+   * Warning - yellow
+   */
+  public static enum AlertType {
+    Warning
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00115)